ui/public/index.html

```diff
@@ -24,14 +24,21 @@
     <!--
       Add the Roboto font required by Material UI.
     -->
-    <link rel="stylesheet"
-    href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Montserrat"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
     />
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-<link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
-
-    <title>Agoric Core HQ</title>
 
+    <title>Agoric Core Economy</title>
```

Comment:
how is changing the title part of fixing layout issues or fixing the "No vaults" thing?

Again, a separate commit would help.

But I'm not sure I'm in a good position to speak for the team on what the title should be.

Reply:
an alternative to a separate commit is something in CONTRIBUTING.md about the coding style we're using. Or better yet: a `yarn lint:fix` script that automates it.

Reply:
As discussed, this is already reflected in the version deployed to https://treasury.agoric.app/ which seems to be based on https://github.com/Agoric/dapp-treasury/tree/beta. I'm not sure why we didn't change it in `main` as well already, but either way we will revisit the title/branding question in a future sync.